### PR TITLE
Temporally disable ansible-test package on SLE

### PR DESCRIPTION
**Pull request by pdostal**
- Previous pull request: #16933

* **Package ansible-test requires Python2.7** [poo#128300](https://progress.opensuse.org/issues/128300), [bsc#1210875](https://bugzilla.suse.com/show_bug.cgi?id=1210875)
    Enforcing `python3` in `ansible.cfg` on **SLE 15-SP5**.
* **Nothing provides 'python3-virtualenv** [poo#128327](https://progress.opensuse.org/issues/128327), [bsc#1210876](https://bugzilla.suse.com/show_bug.cgi?id=1210876)
    Skipping `ansible-test` on **SLE 15-SP54**.
- Verification run: [Leap 15.5](https://pdostal-server.suse.cz/tests/4373), [SLE 15-SP5](https://pdostal-server.suse.cz/tests/4372), [SLE 15-SP4](https://pdostal-server.suse.cz/tests/4374)

### CODE REVIEW

**Comment by Vogtinator** [comment]
Why not apply @rfan1's workaround instead?

**Comment by pdostal** [comment]
> Why not apply @rfan1's workaround instead?

Done.

File: tests/console/ansible.pm [hunk 1]
```diff
@@ -132,8 +138,12 @@ sub run {
     assert_script_run "ansible-playbook -i hosts main.yaml --check $skip_tags", timeout => 300;
 
     # Run the ansible sanity test
-    script_run 'ansible-test --help';
-    assert_script_run 'ansible-test sanity';
+    if (script_run('ansible-test')) {
```
Comment (Vogtinator) [inline tests/console/ansible.pm:141]: In which cases does this fail?

File: tests/console/ansible.pm [hunk 2]
```diff
@@ -25,8 +25,11 @@ use transactional qw(trup_call check_reboot_changes);
 # git-core needed by ansible-galaxy
 # sudo is used by ansible to become root
 # python3-yamllint needed by ansible-test
-my $pkgs = 'ansible ansible-test git-core';
+my $pkgs = 'ansible git-core';
 $pkgs .= ' python3-yamllint' unless is_alp;
+# https://bugzilla.suse.com/show_bug.cgi?id=1210875 Package ansible-test requires Python2.7
+# https://bugzilla.suse.com/show_bug.cgi?id=1210876 Nothing provides 'python3-virtualenv'
+$pkgs .= ' ansible-test' unless is_sle('>=15-SP5');
```
Comment (Vogtinator) [inline tests/console/ansible.pm:32]: I assume Leap is affected the same way?

Comment (pdostal) [reply tests/console/ansible.pm]: It's not.

File: tests/console/ansible.pm (hunk 1 above)
Comment (pdostal) [reply tests/console/ansible.pm]: I improved this. Please re-check.

**Review by Vogtinator — APPROVED** [review]

**Pull request merged**